packages/scandipwa/src/component/ProductTabs/ProductTabs.style.scss

```diff
@@ -35,3 +43,23 @@
         }
     }
 }
+
+.ProductTabsContent {
+    overflow: hidden;
+    height: 0;
+    transition: height var(--transition-speed) ease;
+
+    .fadeIn {
```

Comment:
yeah, as mentioned above, better to stick with the ScandiPWA styling standard and use the `mods` prop for this.